fabric-command-api-v1/src/main/resources/fabric-command-api-v1.mixins.json

```diff
@@ -3,7 +3,8 @@
   "package": "net.fabricmc.fabric.mixin.command",
   "compatibilityLevel": "JAVA_8",
   "mixins": [
-    "MixinCommandManager"
+    "MixinCommandManager",
+    "MixinMinecraftDedicatedServer"
```

Comment:
Should we mixin this on client

Reply:
Yes, this follows how it was done before

Reply:
Agreed, now whole dedi server is shipped on client so this sounds good

Reply:
it may not be ideal, but this needed fixing now, id be happy to look at a follow up PR that improves this.